apache-rat-core/src/main/java/org/apache/rat/configuration/builders/NotBuilder.java

```diff
@@ -19,24 +19,23 @@
 package org.apache.rat.configuration.builders;
 
 import org.apache.rat.ConfigurationException;
-import org.apache.rat.analysis.IHeaderMatcher;
 import org.apache.rat.analysis.matchers.NotMatcher;
 
 /**
  * A builder for the NotMatcher.
  */
 public class NotBuilder extends ChildContainerBuilder {
-    
+
     @Override
-    public IHeaderMatcher build() {
+    public NotMatcher build() {
         if (children.size() != 1) {
             throw new ConfigurationException("'not' type matcher requires one and only one enclosed matcher");
         }
         return new NotMatcher(getId(), children.get(0).build());
     }
-    
+
     @Override
     public String toString() {
-        return String.format( "NotBuilder: %s", !children.isEmpty() ? children.get(0) : null );
+        return String.format("NotBuilder: %s", !children.isEmpty() ? children.get(0) : null);
```

Comment:
What is the semantics of an empty NotBuilder? Is it "not empty"? 
Personally I'd expect an empty notBuilder to be in an illegal state